packages/nuxt3/src/app/nuxt.ts

```diff
@@ -25,13 +25,13 @@ export interface RuntimeNuxtHooks {
   'meta:register': (metaRenderers: Array<(nuxt: NuxtApp) => NuxtMeta | Promise<NuxtMeta>>) => HookResult
 }
 
-export interface NuxtApp {
+export interface _NuxtApp {
```

Comment:
To avoid confusion of usage, maybe keep it unexported?

```suggestion
interface _NuxtApp {
```